tests/testapp/tests/test_filter_classes.py

```diff
@@ -63,11 +61,9 @@ def test_browsable_api_filter_owner(self):
             user=operator, is_admin=True, organization=self._get_org('org_a')
         )
         token = self._obtain_auth_token()
-        auth = dict(HTTP_AUTHORIZATION=f'Bearer {token}')
+        url = f'{reverse("test_books_list_manager_view", args=(self.shelf_a.id,))}'
```

Comment:
Hey, why was this changed?

Reply:
I rebased this branch last time. I will remove this commit for here.

Reply:
Okay, thanks